packages/website/src/widgets/radius-box/utils/formatDecimals.ts

```diff
@@ -0,0 +1,14 @@
+export function formatDecimals(value: string | number) {
```

Comment:
Does this need to support both `string` and `number`? The function name makes me think it should just require a `number`.

Reply:
I did with both because 

1. In the Class function we convert the decimals from number -> number
https://github.com/hackforla/lucky-parking/blob/5a04ae4647faa88a4484c1ff6b19895bb195e146/packages/website/src/widgets/radius-box/utils/unit-convertor.ts#L38-L46

2. In the UI. input returns string type on value change. When we change input, I transform the string to  number, convert it, then transform back to string. 
https://github.com/hackforla/lucky-parking/blob/5a04ae4647faa88a4484c1ff6b19895bb195e146/packages/website/src/widgets/radius-box/radius-tool.tsx#L40-L44

At the parent level we can transform string input -> number
convert the decimals
then convert back to string

This would simplify the function. What are your thoughts?

Reply:
Yeah, let's rely on the function contract and have the caller pass in the correct parameters. In this case, we don't need to guard against everything.